src/util/triplea/MapXMLCreator/MapPropertiesPanel.java

```diff
@@ -0,0 +1,360 @@
+package util.triplea.MapXMLCreator;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.FocusEvent;
+import java.awt.event.FocusListener;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.io.File;
+
+import javax.swing.AbstractAction;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+import util.image.FileOpen;
+
+/**
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+public class MapPropertiesPanel {
+	
+	public static void layout(final MapXMLCreator mapXMLCreator, final JPanel m_stepActionPanel) {
+
+		if (!MapXMLHelper.s_xmlStrings.containsKey("info_@name"))
+		{
+			for (final String newKey : new String[] { "info_@name", "info_@version" })
+			{
+				MapXMLHelper.s_xmlStrings.put(newKey, "");
+			}
+		}
+		
+		if (MapXMLHelper.s_resourceList.isEmpty())
+			MapXMLHelper.s_resourceList.add("");
+
+		final JTextField tMapName = new JTextField(MapXMLHelper.s_xmlStrings.get("info_@name"));

```

Comment:
Actually in this case it makes a lot of sense: On the one hand the labels are grouped together and on the other hand you can see immediately what should be available in the panel.